pkg/util/aad/aad.go

```diff
@@ -18,6 +18,41 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/refreshable"
 )
 
+func refreshContext(authorizer *refreshable.Authorizer, ctx *context.Context, done *bool, log *logrus.Entry) error {
```

Comment:
the context usually goes first https://pkg.go.dev/context, also there is no need to use a pointer there
Do you need a pointer to the authorizer ? It seems to be an interface, and the pointer seems unnecessary here